test/models.test.js

```diff
@@ -50,10 +52,74 @@ describe('collections', function () {
 
 describe('models', function() {
   describe('centre', function() {
+    beforeEach(function() {
+      var io;
+
+      this.server = new socket();
+      this.fakeio = function(server) {
+        return function() {
+          return server.socketClient;
+        }
+      };
+
+      io = sailsIOClient(this.fakeio(this.server));
+      io.sails.autoConnect = false;
+
+      this.socket0 = io.sails.connect();
+    });
     it('should be instantiated', function() {
       var model = new models.Centre();
 
       expect(model).not.to.be(undefined);
```

Comment:
In Stephen's absence, can you clarify your comment @chrisns ? Are you referring to the last four lines of the above snippet?

Something like the following? Feel free to correct me!

it('should be instantiated', function(done,error) {
       var model = new models.Centre();
       expect(model).to.be.ok();
       expect(model).not.to.be(undefined);
...`